torch/_inductor/codegen/triton.py

```diff
@@ -1217,11 +1217,10 @@ def tan(x):
     @staticmethod
     @maybe_upcast_float32()
     def tanh(x):
-        if config.use_fast_math and torch.version.hip:
-            if get_triton_version() > (3, 4):
-                return f"libdevice.fast_tanhf({x})"
-            else:
-                return f"libdevice.tanh({x})"
+        # On ROCm, always use fast_tanhf
+        # Requires ROCm fork of Triton 3.3, 3.4, 3.5 or upstream Triton 3.6+
+        if torch.version.hip:
```

Comment:
2.7 uses 3.3 IIUC

We should at least support 3.2 in 2.7, so lets conditionalise on triton > (3,3) if 3.3 supports this.